server/game/cards/02_SHD/events/Outflank.ts

```diff
@@ -0,0 +1,23 @@
+import { EventCard } from '../../../core/card/EventCard';
+
+export default class Outflank extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '0802973415',
+            internalName: 'outflank',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Attack with a unit',
+            initiateAttack: {},
+            then: {
+                title: 'Attack with another unit',
```

Comment:
Need to add the filter so that the same unit can't be chosen twice, in case the first unit becomes readied again (e.g. Mace Windu ability). See the impl of Maximum Firepower for how to do this with a `then` ability.